lib/pr_checker.js

```diff
@@ -168,6 +168,25 @@ class PRChecker {
       }
     }
 
+    if (isFastTracked) {
+      const comment = this.comments.find((c) =>
+        c.bodyText.startsWith('Fast-track'));
+      if (!comment) {
+        cli.error('Unable to find the fast-track request comment.');
+        return false;
+      }
+      const [, requestor] = comment.bodyText.match(/^Fast-track has been requested by @(.+?)\. Please 👍 to approve\.$/);
+      const collaborators = [...this.data.collaborators.values()]
+        .map((c) => c.login);
+      if (!comment.reactions.nodes.some((r) =>
+        r.user.login !== requestor &&
+        collaborators.includes(r.user.login))) {
+        cli.error('The fast-track request requires' +
+                  " at least one collaborator's 👍.");
```

Comment:
Fast tracking requires two collaborator approvals.

> The pull request can be fast-tracked if two collaborators approve the fast-tracking request. To land, the pull request itself still needs two collaborator approvals and a passing CI.



Ref: https://github.com/nodejs/node/blob/86099a375af95f519ae699b68ce101aa20236c5f/doc/guides/collaborator-guide.md#waiting-for-approvals

Would it be possible to implement logic such that it checks to see if the person that requested fast-tracking is also the author of the PR and, if so, checks for two collaborator fast-track 👍 reactions, and otherwise accepts one collaborator fast-track 👍 reaction as long as it's not a 👍 from either the PR author or the collaborator proposing fast-tracking?

In other words:

```
If fast-track requester is PR author:
    Require 2 👍 reactions from collaborators who are not the PR author
else:
    Require 1 👍 reaction from a collaborator who is not the PR author or the collaborator who requested fast-tracking
```